.travis.yml

```diff
@@ -29,4 +29,4 @@ before_script:
   - cd ~/builds/ss
 
 script:
-  - phpunit silverstripe-lumberjack/tests/
\ No newline at end of file
+  - vendor/bin/phpunit lumberjack/tests/
```

Comment:
There is benefit in keeping it prefixed with silverstripe-. You can just add silverstripe-* to your .gitignore which means people don't keep forgetting to add modules. It was a workaround as we don't use vendor/ or have a dedicated module folder.